doc/HotWordBoosting-Examples.rst

```diff
@@ -0,0 +1,55 @@
+Hot-word boosting API Usage example
+===================================
+
+With DeepSpeech 0.9 release a new API feature was introduced that allows boosting probability from the scorer of given words. It is exposed in all bindings (C, Python, JS, Java and .Net). 
+
+Currently, it provides three methods for the Model class:
+
+- ``AddHotWord(word, boost)``
+- ``EraseHotWord(word)`` 
+- ``ClearHotWords()``
+
+Exact API binding for the language you are using can be found in API Reference.
+
+Example 
+-------
+To use hot-word boosting just add hot-words of your choice before performing an inference to a ``Model``. You can also erase boosting of a chosen word or clear it for all hot-words.
+
+.. code-block:: python
+
+	ds = Model(args.model)
+	...
+	ds.addHotWord(word, boosting)
+	...
+	print(ds.stt(audio))
+	
+Adding boost value to a word many times or erasing hot-word without previously boosting it results in an error.
+
+General usage
+-------------
+
+It is worth noting that boosting non-existent words in scorer (mostly proper nouns) or a word that share no phonetic prefix with other word in the input audio don't change the final transcription. Additionally, hot-word that has a space will not be taken into consideration, meaning that combination of words can not be boosted and each word must be added as hot-word separately. 
+
+
+Positive value boosting
```

Comment:
Should this be done on the API part as well? I thought we documented that, but checking the C API, it's not there: https://deepspeech.readthedocs.io/en/v0.9.3/C-API.html#_CPPv413DS_AddHotWordP10ModelStatePKcf

Reply:
This should be adapted here:
 - https://github.com/mozilla/DeepSpeech/blob/962a117f7ed5720435904e3ac864cc8420256ca1/native_client/deepspeech.h#L163-L175
 - https://github.com/mozilla/DeepSpeech/blob/962a117f7ed5720435904e3ac864cc8420256ca1/native_client/javascript/index.ts#L191-L199
etc. etc. for each binding

Reply:
Okay sure, I'll adapt it for each API bindings, makes more sense and it will be more readable

Reply:
Great